tools/src/test/python/dlpx/virtualization/_internal/test_cli.py

```diff
@@ -319,11 +319,11 @@ def test_file_not_exist(plugin_config_file):
 
         assert result.exit_code == 2
         assert result.output == (u'Usage: delphix-sdk build [OPTIONS]'
-                                 u'\nTry "delphix-sdk build -h" for help.'
+                                 u'\nTry \'delphix-sdk build -h\' for help.'
                                  u'\n'
-                                 u'\nError: Invalid value for "-c" /'
-                                 u' "--plugin-config": File'
-                                 u' "/not/a/real/file/plugin_config.yml"'
+                                 u'\nError: Invalid value for \'-c\' /'
+                                 u' \'--plugin-config\': File'
+                                 u' \'/not/a/real/file/plugin_config.yml\''
```

Comment:
nit: I think our standard is if you're using single quotes in the string, we can then use double quotes to start the string (so that we don't have to use backslash):
so it would be something like:
```
assert result.output == (u"Usage: delphix-sdk build [OPTIONS]"
                                         u"\nTry 'delphix-sdk build -h' for help."
                                         u"\n"
                                         u"\nError: Invalid value for '-c' /"
                                         u"'--plugin-config': File"
                                         u" '/not/a/real/file/plugin_config.yml'"